src/main/java/frc/robot/Robot.java

```diff
@@ -79,11 +79,9 @@ public void disabledPeriodic() {
      */
     @Override
     public void autonomousInit() {
-
-//        robotContainer.resetModules();
-
+        // TODO: why is this here? A robot may not have vision. The container may not have the method.
```

Comment:
This sets the enabled variable in Drivetrain so that it doesn't use vision in auto if we disable it in the constants. If the robot doesn't have vision, this variable will do nothing.

Reply:
It should not be in Robot.java. Robot.java should not know about vision or subsystems.

Reply:
Where should it be instead? This needs to be called at the start of auto, and the only class that knows when that is is Robot.

Reply:
It should be called somewhere else. If needed in an auto routine, then it can be called there and disabled there. Currently it is not disabled, so the behavior is not clean.

In any event, the call is there now, so it should not hold up the PR.